packages/termux-wsi-layer/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.29)
+project(termux-wsi-layer C)
+
+set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_C_STANDARD 11)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(X11 REQUIRED x11 x11-xcb xcb xcb-dri3 xcb-present)
+
+add_library(termux-wsi-layer SHARED egl.c window.c sync.c)
+set_target_properties(termux-wsi-layer PROPERTIES PREFIX "")
+target_include_directories(termux-wsi-layer PRIVATE include)
+target_compile_options(termux-wsi-layer PRIVATE "-D__ANDROID_UNAVAILABLE_SYMBOLS_ARE_WEAK__" ${X11_CFLAGS})
```

Comment:
Should get rid of `__ANDROID_UNAVAILABLE_SYMBOLS_ARE_WEAK__`. Better to fix compile errors then to fix runtime errors.

Reply:
The project uses AHardwareBuffer feature which is available on android 8+, but termux targets lower version of Android. Of course I can try to declare weak symbols manually and check if they are null or undefined using dladdr, but that seems to be a bit more complicated. 

I am planning to write some postinst check which will disable the ICD in the case if vendor changed ANativeWindow signature or Android version is too low.